### PR TITLE
Init CustomerExportImport project

### PR DESCRIPTION
**Pull request by trueboroda**

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-7399 has been created

File: src/VirtoCommerce.CustomerExportImportModule.Core/ModuleConstants.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+namespace VirtoCommerce.CustomerExportImportModule.Core
+{
+    public static class ModuleConstants
+    {
+        public static class Features
+        {
+            public const string CustomerExportImport = "CustomerExportImport";
+
+            public const string DemoFeaturesPermission = "features:developer";
```
Comment (pushnitsa) [inline src/VirtoCommerce.CustomerExportImportModule.Core/ModuleConstants.cs:12]: What is it? Here's need specify feature flag, not permission

File: src/VirtoCommerce.CustomerExportImportModule.Core/ModuleConstants.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+                public const string ImportAccess = "customer:import";
+
+                public static string[] AllPermissions { get; } = { ExportAccess, ImportAccess };
+            }
+        }
+
+        public static class Settings
+        {
+            public const int PageSize = 50;
```
Comment (pushnitsa) [inline src/VirtoCommerce.CustomerExportImportModule.Core/ModuleConstants.cs:29]: What for we need page size setting moved to settings class?

File: src/VirtoCommerce.CustomerExportImportModule.Web/Module.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                    Name = x
+                }).ToArray());
+
+            var featureStorage = appBuilder.ApplicationServices.GetService<IFeatureStorage>();
+            featureStorage.TryAddFeatureDefinition(ModuleConstants.Features.CustomerExportImport, ModuleConstants.Features.DemoFeaturesPermission);
+
+            // ensure that all pending migrations are applied
+            using (var serviceScope = appBuilder.ApplicationServices.CreateScope())
+            {
+                using (var dbContext = serviceScope.ServiceProvider.GetRequiredService<VirtoCommerceCustomerExportImportDbContext>())
```
Comment (pushnitsa) [inline src/VirtoCommerce.CustomerExportImportModule.Web/Module.cs:69]: You can union this using statements

**Review by pushnitsa — APPROVED** [review]

**Review by Pav-el — APPROVED** [review]

**Pull request merged**